app/controllers/records_controller.rb

```diff
@@ -0,0 +1,43 @@
+class RecordsController < ApplicationController
+
+    before_action :set_user
+    before_action :set_user_record, only: [:show, :update, :destroy ]
+
+    def index
+        json_response(@user.records)
+    end
+
+    def show
+        json_response(@record)
+    end
+
+    def create
+        @user.records.create!(record_params)
+        json_response(@user, :created )
+    end 
+
+    def update
+        @record.update(record_params)
+        head :no_content
+    end
+
+    def destroy
+        @record.destroy 
+        head :no_content
+    end
+
+    private 
+
+    def record_params 
+        params.permit( :user_id, :specialist_id, :description, :diagnosis)
```

Comment:
Lets use `params.require(:record).permit( :user_id, :specialist_id, :description, :diagnosis)`

Reply:
thanks for that @wathigo, worked on it.